DOC-1720: Edit down descriptions for databases/durability-ha index page (#2341)
RedisLabs/redislabs-docs: content/rs/databases/durability-ha/_index.md

## [Replication]({{<relref "/rs/databases/durability-ha/replication.md">}})

When you replicate your database, each database instance (shard) is copied one or more times. Your database will then have one primary shard and one or more replica shards. When a primary shard fails, Redis Enterprise automatically promotes the replica shard to primary. 

## [Clustering]({{<relref "/rs/databases/durability-ha/clustering.md">}})

> Clustering (or sharding) breaks your database into individual instances (shards) that are then spread across several nodes. Clustering allows you to scale your database as you resources to your cluster and stops node failure from causing availability loss.

Reviewer: ```suggestion
Clustering (or sharding) breaks your database into individual instances (shards) and spreads them across several nodes. Clustering lets you add resources to your cluster to scale your database and prevents node failures from causing availability loss.
```

## [Database persistence]({{<relref "/rs/databases/configure/database-persistence.md">}})

> Database persistence gives your database durability against process or server failures by saving data information to disk at set intervals.

Reviewer: ```suggestion
Database persistence gives your database durability against process or server failures by saving data to disk at set intervals.
```

## [Active-Active geo-distributed replication]({{<relref "/rs/databases/active-active/_index.md">}})

> Redis Enterprise Active-Active distributes your replicated data across multiple nodes and availability zones. This increases the durability of your database by reducing the likelihood of data or availability loss. It also reduces the latency of your data access.

Reviewer: ```suggestion
Active-Active Redis Enterprise databases distribute your replicated data across multiple nodes and availability zones. This increases the durability of your database by reducing the likelihood of data or availability loss. It also reduces data access latency.
```

## [Rack-zone awareness]({{<relref "/rs/clusters/configure/rack-zone-awareness.md">}})

> Rack-zone awareness maps each node in your Redis Enterprise cluster to a physical rack or logical zone. The cluster then uses that information to distribute primary shards and their replica shards in different racks or zones. This ensures data availability if there is rack or zone failure.

Reviewer: ```suggestion
Rack-zone awareness maps each node in your Redis Enterprise cluster to a physical rack or logical zone. The cluster uses this information to distribute primary shards and their replica shards in different racks or zones. This ensures data availability if a rack or zone fails.
```

## [Discovery service]({{<relref "/rs/databases/durability-ha/discovery-service.md">}})

> The discovery service provides an IP-based connection management service used when connecting to Redis Enterprise Software databases. It allows your application to discover which node hosts the database endpoint. The API used for discovery service is compliant with the Redis Sentinel API.

Reviewer: ```suggestion
The discovery service provides an IP-based connection management service used when connecting to Redis Enterprise Software databases. It lets your application discover which node hosts the database endpoint. The discovery service API complies with the [Redis Sentinel API](https://redis.io/docs/management/sentinel/#sentinel-api).
```
